File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Invs/INV4001U00Grd4001Info.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Invs
{
    public class INV4001U00Grd4001Info
    {
        private String _pkinv4001;
        private String _churiDate;
        private String _importTime;
        private String _churiBuseo;
        private String _ipgoType;
        private String _importCode;
        private String _churiSeq;
        private String _jaeryoGubun;
        private String _remark;
        private String _inOutGubun;
        private String _ipchulType;
        private String _inputUser;
        private String _rowState;

        public String Pkinv4001
        {
            get { return this._pkinv4001; }
            set { this._pkinv4001 = value; }
        }

        public String ChuriDate
        {
            get { return this._churiDate; }
            set { this._churiDate = value; }
        }

        public String ImportTime
        {
            get { return this._importTime; }
            set { this._importTime = value; }
        }

        public String ChuriBuseo
        {
            get { return this._churiBuseo; }
            set { this._churiBuseo = value; }
        }

        public String IpgoType
        {
            get { return this._ipgoType; }
            set { this._ipgoType = value; }
        }

        public String ImportCode
        {
            get { return this._importCode; }
            set { this._importCode = value; }
        }

        public String ChuriSeq
        {
            get { return this._churiSeq; }
            set { this._churiSeq = value; }
        }

        public String JaeryoGubun
        {
            get { return this._jaeryoGubun; }
            set { this._jaeryoGubun = value; }
        }

        public String Remark
        {
            get { return this._remark; }
            set { this._remark = value; }
        }

        public String InOutGubun
        {
            get { return this._inOutGubun; }
            set { this._inOutGubun = value; }
        }

        public String IpchulType
        {
            get { return this._ipchulType; }
            set { this._ipchulType = value; }
        }

        public String InputUser
        {
            get { return this._inputUser; }
            set { this._inputUser = value; }
        }

        public String RowState
        {
            get { return this._rowState; }
            set { this._rowState = value; }
        }

        public INV4001U00Grd4001Info() { }

        public INV4001U00Grd4001Info(String pkinv4001, String churiDate, String importTime, String churiBuseo, String ipgoType, String importCode, String churiSeq, String jaeryoGubun, String remark, String inOutGubun, String ipchulType, String inputUser, String rowState)
        {
            this._pkinv4001 = pkinv4001;
            this._churiDate = churiDate;
            this._importTime = importTime;
            this._churiBuseo = churiBuseo;
            this._ipgoType = ipgoType;
            this._importCode = importCode;
            this._churiSeq = churiSeq;
            this._jaeryoGubun = jaeryoGubun;
            this._remark = remark;
            this._inOutGubun = inOutGubun;
            this._ipchulType = ipchulType;
            this._inputUser = inputUser;
            this._rowState = rowState;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/invs/INV4001U00Grd4001Info.java
package nta.med.data.model.ihis.invs;

import java.sql.Date;
import java.sql.Timestamp;

public class INV4001U00Grd4001Info {
	 private Double pkinv4001     ;
	 private Timestamp churiDate    ;
	 private String importTime;
	 private String churiBuseo   ;
	 private String ipgoType     ;
	 private String importCode;
	 private Double churiSeq     ;
	 private String jaeryoGubun  ;
	 private String remark        ;
	 private String inOutGubun  ;
	 private String ipchulType   ;
	 private String rowState     ;
	public INV4001U00Grd4001Info(Double pkinv4001, Timestamp churiDate, String importTime, String churiBuseo,
			String ipgoType, String importCode, Double churiSeq, String jaeryoGubun, String remark, String inOutGubun,
			String ipchulType) {
		super();
		this.pkinv4001 = pkinv4001;
		this.churiDate = churiDate;
		this.importTime = importTime;
		this.churiBuseo = churiBuseo;
		this.ipgoType = ipgoType;
		this.importCode = importCode;
		this.churiSeq = churiSeq;
		this.jaeryoGubun = jaeryoGubun;
		this.remark = remark;
		this.inOutGubun = inOutGubun;
		this.ipchulType = ipchulType;
	}
	public Double getPkinv4001() {
		return pkinv4001;
	}
	public void setPkinv4001(Double pkinv4001) {
		this.pkinv4001 = pkinv4001;
	}
	public Timestamp getChuriDate() {
		return churiDate;
	}
	public void setChuriDate(Timestamp churiDate) {
		this.churiDate = churiDate;
	}
	public String getImportTime() {
		return importTime;
	}
	public void setImportTime(String importTime) {
		this.importTime = importTime;
	}
	public String getChuriBuseo() {
		return churiBuseo;
	}
	public void setChuriBuseo(String churiBuseo) {
		this.churiBuseo = churiBuseo;
	}
	public String getIpgoType() {
		return ipgoType;
	}
	public void setIpgoType(String ipgoType) {
		this.ipgoType = ipgoType;
	}
	public String getImportCode() {
		return importCode;
	}
	public void setImportCode(String importCode) {
		this.importCode = importCode;
	}
	public Double getChuriSeq() {
		return churiSeq;
	}
	public void setChuriSeq(Double churiSeq) {
		this.churiSeq = churiSeq;
	}
	public String getJaeryoGubun() {
		return jaeryoGubun;
	}
	public void setJaeryoGubun(String jaeryoGubun) {
		this.jaeryoGubun = jaeryoGubun;
	}
	public String getRemark() {
		return remark;
	}
	public void setRemark(String remark) {
		this.remark = remark;
	}
	public String getInOutGubun() {
		return inOutGubun;
	}
	public void setInOutGubun(String inOutGubun) {
		this.inOutGubun = inOutGubun;
	}
	public String getIpchulType() {
		return ipchulType;
	}
	public void setIpchulType(String ipchulType) {
		this.ipchulType = ipchulType;
	}
	public String getRowState() {
		return rowState;
	}
	public void setRowState(String rowState) {
		this.rowState = rowState;
	}
	
}
